.github/wiki/Test the apply action on development.md

```diff
@@ -0,0 +1,47 @@
+# Test the apply action on development
```

Comment:
a small concern, should we name this file as 
`Test-The-Apply-Action-On-Development.md`

Reply:
Fixed in ddc985a